Mozzi.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Mozzi.h
+ *
+ * Copyright 2023, Thomas Combriat and the Mozzi team
```

Comment:
Should probably be edited with the correct surname ;)

Reply:
Will do ;-). Although I'll change the first name too, claiming Mozzi.h itself would seems a little over the top.